setup.cfg

```diff
@@ -51,6 +51,7 @@ install_requires =
     rebasehelper
     munch
     Flask
+    packaging
```

Comment:
also update spec file please

Reply:
whoops, my bad, we are using the requirements generators, so we don't need to update spec

Reply:
At least I know too now. I actually forgot to add there.